NGitLab/Impl/PipelineClient.cs

```diff
@@ -48,14 +60,31 @@ public Job[] GetJobs(int pipelineId)
         }
 
         public IEnumerable<Job> GetJobs(PipelineJobQuery query)
+        {
+            var url = CreateGetJobsUrl(query);
+
+            // For some reason GitLab returns the jobs in the reverse order as
+            // they appear in their UI. Here we reverse them!
+            return _api.Get().GetAll<Job>(url).Reverse();
+        }
+
+        public GitLabCollectionResponse<Job> GetJobsAsync(PipelineJobQuery query)
         {
             var url = $"{_pipelinesPath}/{query.PipelineId.ToStringInvariant()}/jobs";
             url = Utils.AddParameter(url, "scope", query.Scope);
             url = Utils.AddParameter(url, "include_retried", query.IncludeRetried);
 
             // For some reason GitLab returns the jobs in the reverse order as
             // they appear in their UI. Here we reverse them!
-            return _api.Get().GetAll<Job>(url).Reverse();
+            return _api.Get().GetAllAsync<Job>(url);
```

Comment:
Missing call to `Reverse()`

Reply:
I don't want to add the `Reverse` because:
- `Reverse` needs to get all results from the API before returning the first result to the caller
- NGitLab should just call the GitLab API, and should avoid changing the result

I think the non-async method should not have the `Reverse` call, but I don't know how people is using this method

Reply:
Ok, then you need to remove the misleading comment that precedes the call.

You should also add a `<summary>` or `<remarks>` for the `GetJobsAsync` method, indicating that unlike the sync version it does not reverse the order.